Jagati.cmake

```diff
@@ -1991,13 +1991,21 @@ macro(EmitExceptionSource)
     set(NamespaceSourceHeader "\
 #include \"DataTypes.h\"\n\
 #include \"MezzException.h\"\n\n\
+SAVE_WARNING_STATE\n\n\
 namespace Mezzanine\n{\n\
 namespace Exception\n{\n\
 \n"
     )
 
     set(NamespaceHeader "\
 #include \"DataTypes.h\"\n\n\
+SAVE_WARNING_STATE\n\
+SUPPRESS_CLANG_WARNING(\"-Winconsistent-missing-destructor-override\")\n\
+#ifdef __clang_major__\n\
+#if __clang_major__ >= 11\n\
```

Comment:
I am assuming if the Clang compiler version is at least version 11, line 2006 suppresses the warning suggesting a different destructor to be used?

Reply:
Yes. Pretty much exactly.

I have Clang 10 on my system, the default that ships with my OS. But Fedora in our Jenkins setup has 11 and it has this warning. The option to suppress in invalid in clang 10 and break my build, So this conditionally surpresses it only when nothing else would break.